webapp/src/main/java/com/plhal/ares/webapp/controller/CompanyController.java

```diff
@@ -13,37 +12,40 @@
 @Controller
 public class CompanyController {
 
-	@Autowired
-	private DataService dataService;
+    private final DataService dataService;
 
-	// Nastaví mapování pro úvodní stránku
-	@GetMapping("/")
-	public String uvodniStranka() {
+    public CompanyController(DataService dataService) {
```

Comment:
Tady jen poznamka - je to v poradku a funkcni. Ta trida ( i pres absenci `autowired` funguje "automagicky"). Trida obsahuje anotaci a tak by zde melo byt `autowired`, ale u parametru v konstruktoru to ted neni treba (tusim od springu 4.x). Nicmene kdybyste to vkladal pres `set` metodu, tak by tam to `autowired` muselo byt.